src/mongo/Config.class.php

```diff
@@ -2026,6 +2064,48 @@ public function getTransactionLogDatabase($readPreference = \MongoClient::RP_PRI
     }
 
     /**
```

Comment:
What's the ETA on TARL migrating to PHP 5.4+?  Perhaps it makes more sense to make 'composite-ness' a trait.